### PR TITLE
Add latest activity timestamp to wallet state

### PR DESCRIPTION
**Pull request by aonyshchenko-strike**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Bug fix breaking (fix that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] New feature breaking (change which adds functionality and causes existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/handlers/migrate_handler.rs [hunk 1]
```diff
@@ -38,14 +33,15 @@ fn migration_test(source: &[u8], destination: &mut [u8], rent_return: &Pubkey) {
         config_approvers: source_account.config_approvers,
         balance_accounts: source_account.balance_accounts,
         dapp_book: source_account.dapp_book,
+        latest_activity_at: 0,
     };
     Wallet::pack(destination_account, destination).unwrap();
 }
 
-const V2_WALLET_SIZE: usize = 8694;
+const V3_WALLET_SIZE: usize = 9462;
```
Comment (bholzman) [inline src/handlers/migrate_handler.rs:41]: we are still on v2 in prod, so we should leave that migration in for now

**Review by bholzman — COMMENTED** [review]
Looks great!

File: src/handlers/migrate_handler.rs (hunk 1 above)
Comment (aonyshchenko-strike) [reply src/handlers/migrate_handler.rs:47]: Ok, I've brought that migration back.

**Review by bholzman — APPROVED** [review]
Looks great!

**Pull request merged**